src/content/documentation/enterprise/enterprise-policies-that-impact-extensions.md

```diff
@@ -40,7 +40,7 @@ These policies control whether users can install, disable, or remove web extensi
   - removed when the policy is deployed, specified by extension ID.
   - locked when the policy is deployed, so the user cannot disable or remove the extension, which are specified by extension ID.
 
-- **ExtensionUpdate** – This policy provides for the setting of the configuration (about:config) option `extensions.update.enabled`. This enables you to determine whether extensions are updated when Firefox detects updates. If you choose to disable automatic updates, you can use the **Extensions** policy to push web extension updates.
+- **ExtensionUpdate** – This policy provides for the setting of the configuration (`about:config`) option `extensions.update.enabled`. This enables you to determine whether extensions are updated when Firefox detects updates. If you choose to disable automatic updates, you can use the **Extensions** policy to push web extension updates.
```

Comment:
There's a couple here that don't have backticks.

Reply:
* “A couple” of what…?
* “that don’t have”
  - …and _should_ have backticks?
  - …that have backticks, but should _not_?

Reply:
You've wrapped the other policy terms below in backticks, but the two above this comment don't have backticks.

Reply:
Addressed in b91ffd3.  (I _think_ I interpreted your feedback correctly.  Happy to adjust this if I didn’t.)